index_quantized.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.css">
+    <meta charset="UTF-8" />
+    <link rel="stylesheet" href="style.css" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="jason.css">
+    <meta property="og:title" content="Portable HNSW">
+    <meta property="og:description" content="Use natural language to search without a server">
+    <meta property="og:type" content="website">
+    <meta property="og:url" content="https://jasonjmcghee.github.io/portable-hnsw/">
+
+    <title>Portable HNSW</title>
+    <style>
+        .container {
+            padding: 16px;
+            display: flex;
+            justify-content: center;
+        }
+        .inner {
+            display: flex;
+            justify-content: center;
+            flex-direction: column;
+            max-width: 1000px;
+            width: 100%;
+            gap: 20px;
+        }
+        .query {
+            display: flex;
+            justify-content: center;
+            gap: 12px;
+        }
+        #text-input {
+            flex-grow: 1;
+        }
+        .path-input {
+            display: flex;
+            align-items: center;
+            gap: 12px;
+        }
+        #instant-mode {
+          cursor: pointer;
+        }
+        #path {
+            flex-grow: 1;
+        }
+    </style>
+</head>
+
+<body>
+    <main class="container">
+        <div class="inner">
+            <h1 class="title">Use natural language to search 50k blog posts without a server</h1>
+            Or something else if you build your own index and throw in it in a CDN.
+            <a href="https://github.com/jasonjmcghee/portable-hnsw">What is going on here? / Show me the code</a>
+            <i>Too slow for you? I'm probably doing things poorly. PRs welcome!</i>
+            <div class="path-input">
+              <a href="" id="frankenstein">Frankenstein (fastest)</a>
+              <a href="" id="shakespeare">Shakespeare</a>
+              <a href="" id="blogs">50k Blogs Dataset (slowest - ~15 seconds)</a>
+            </div>
+            <div class="path-input">
+                Path: <input type="text" id="path" placeholder="https://jasonjmcghee.github.io/portable-hnsw/frankenstein" value="frankenstein"></input>
+            </div>
+            <div class="path-input">Debounced auto-search<input id="instant-mode" type="checkbox"></input></div>
+            <div>
+                <div class="query">
+                    <textarea id="text-input">falling in love</textarea>
+                    <button disabled id="search">Search</button>
+                </div>

```

Comment:
not trying to be pedantic here, would love to get this merged in! But at the moment this won't work properly according to the instructions here. It will **only** work with quantized indices.

Reply:
I can update the readme and / or this text if you'd like to describe what's going on. lmk!

Reply:
looks good now?

Reply:
updated, looks good now?

Reply:
A month went by - sorry to not engage more, but AFAICT this still won't work properly unless i'm misunderstanding things.

If I'm reading things correctly, it's referencing datasets in this git repo (shakespeare, frakenstein, blogs 50k) that are not quantized. So I believe this will always fail, as they are referencing the original datasets, not quantized versions.

Reply:
hey, hehe no worries I love to brainstorm!, I had almost forgotten what did I write lol. Could you point where is this?

If its the html files, the dataset remains the same, it the query that gets quantized and then is compared with the quantized `.parquet` files from `vevctordb_utils.js`